app/mailers/user_mailer.rb

```diff
@@ -44,51 +44,51 @@ def new_message(user_id,message_id)
     @message = Message.find_by_id(message_id)
     mail(:subject => "openSNP.org: You've got a new mail from #{User.find_by_id(@message.from_id).name}", :to => @user.email)
   end
-  
+
   def new_snp_comment(snp_comment,to_user)
     @user = to_user
     @snp_comment = snp_comment
     mail(:subject => "openSNP.org: You've got a reply to one of your SNP-comments", :to => @user.email)
   end
-  
+
   def new_phenotype_comment(phenotype_comment,to_user)
     @user = to_user
     @phenotype_comment = phenotype_comment
     mail(:subject => "openSNP.org: You've got a reply to one of your phenotype-comments", :to => @user.email)
   end
-  
+
   def new_picture_phenotype_comment(phenotype_comment,to_user)
     @user = to_user
     @phenotype_comment = phenotype_comment
     mail(:subject => "openSNP.org: You've got a reply to one of your phenotype-comments", :to => @user.email)
-  end  
-  
+  end
+
   def new_phenotype(phenotype,user)
     @user = user
     @phenotype = phenotype
     mail(:subject => "openSNP.org: A new phenotype was entered on the platform", :to => @user.email)
   end
-  
+
   def newsletter(user)
     @user = user
-    mail(:subject => "openSNP.org: #thedress, ethics for participant-lead research, research hack days", :to => @user.email)
+    mail(:subject => "openSNP.org: participate in a survey on the experiences of sharing genetic data", :to => @user.email)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.